api/v1alpha1/jindoruntime_types.go

```diff
@@ -153,6 +153,10 @@ type JindoRuntimeSpec struct {
 
 	// +optional
 	LogConfig map[string]string `json:"logConfig,omitempty"`
+
+	// NetworkMode Host or Container
+	// +optional
+	NetworkMode string `json:"networkmode,omitempty"`
```

Comment:
I suggest following the way of placementMode

```golang
// Manage switch for opening Multiple datasets single node deployment or not
	// TODO(xieydd) In future, evaluate node resources and runtime resources to decide whether to turn them on
	// +kubebuilder:validation:Enum=Exclusive;"";Shared
	// +optional
	PlacementMode PlacementMode `json:"placement,omitempty"`
```

Reply:
thx, done